src/framework/preferences/js/Panels.js

```diff
@@ -767,13 +818,25 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         },
         selectors: {
             lineSpace: ".flc-prefsEditor-line-space",
-            label: ".flc-prefsEditor-line-space-label",
+            label: ".flc-prefsEditor-line-space-label"
+        },
+        selectorsToIgnore: ["lineSpace"],
+        protoTree: {
+            label: {messagekey: "lineSpaceLabel"},
+            narrowIcon: {messagekey: "lineSpaceNarrowIcon"},
+            wideIcon: {messagekey: "lineSpaceWideIcon"},
+            multiplier: {messagekey: "multiplier"},
+            lineSpaceDescr: {messagekey: "lineSpaceDescr"}
+        }
+    });
+
+    fluid.defaults("fluid.prefs.defaultLineSpace", {
```

Comment:
Probably better to name these more specifically. For example "fluid.prefs.lineSpace.textfieldSlider" and "fluid.prefs.lineSpace.textfieldButtons"